tutorials-and-examples/hf-tgi/main.tf

```diff
@@ -47,6 +47,10 @@ resource "kubernetes_service" "inference_service" {
 }
 
 resource "kubernetes_deployment" "inference_deployment" {
+  timeouts {
+    create = "30m"
```

Comment:
Does it wait 30m for the deployment to be created or the deployment to be "ready". I think this only needed if it checks for ready deployment

Reply:
Tested without it and got error `Error: Waiting for rollout to finish: 1 replicas wanted; 0 replicas Ready`. So I think it's for the deployment to be "ready".